src/WebJobs.Script/Workers/Rpc/RpcInitializationService.cs

```
@@ -127,79 +98,28 @@ internal async Task InitializeRpcServerAsync()
             }
             catch (Exception grpcInitEx)
             {
-                var hostInitEx = new HostInitializationException($"Failed to start Rpc Server. Check if your app is hitting connection limits.", grpcInitEx);
+                throw new HostInitializationException($"Failed to start Rpc Server. Check if your app is hitting connection limits.", grpcInitEx);
```

Interesting... were we ignoring this exception previously?

---

I assumed this was an oversight and fixed it !

---

@brettsam  - OK to resolve this comment?